contracts/src/v0.8/ccip/offRamp/OffRamp.sol

```diff
@@ -787,10 +788,13 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
     bytes32 rawVs
   ) external {
     CommitReport memory commitReport = abi.decode(report, (CommitReport));
+    DynamicConfig storage dynamicConfig = s_dynamicConfig;
 
     // Verify RMN signatures
-    if (commitReport.merkleRoots.length > 0) {
-      i_rmnRemote.verify(address(this), commitReport.merkleRoots, commitReport.rmnSignatures, commitReport.rmnRawVs);
+    if (!dynamicConfig.isRMNVerificationDisabled) {
```

Comment:
One last plea: why not call this `rmnVerificationEnabled`? Then we avoid the double negative when talking about pretty much everything related to this var in general:

* isRMNVerificationDisabled == true => means that RMN is disabled => means that RMN is not enabled
* isRMNVerificationDisabled == false => means that RMN is not disabled => means that RMN is enabled

It just seems like an extra cognitive overhead that IMO is never worth doing for boolean vars. Always prefer positive connotation to negative IMO.

Reply:
There was a separate thread requesting that the 0 value for this config (false) be the "safe" value (which you supported btw!! 🙃). This would help ensure verification is enabled by default.

Here's what we could do... we could change it to isCommitPermablessed this avoids the double-negative and still has false as the "safe" value. IMHO though, the functionality of that name is less obvious. isRMNVerificationDisabled is pretty obvious about what it does.